statement_test.go

```diff
@@ -430,7 +430,9 @@ func TestBindWithoutResolvedParams(t *testing.T) {
 	// Type without a fallback.
```

Comment:
Let's update the comment, too. Also, can we add another test for a type we do not yet support (e.g., `UNION`, I think)?